Assignment2/derbeire-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,54 @@
+{
+	"@context": {
+		"cla":"http://schema.org/",
+		"xsd":"http://schema.org/xsd",
+		"Class01":"cla:Class01",
+		"Sensor029":"cla:Sensor029",
+		"Computer101":"cla:Computer101",
+		"Measurement8401":"sensor:Measurement8401",
+		"User10A":"computer:User10A",
+		"includes":"cla:includes",
+		"hasMeasurement":"cla:hasMeasurement",
+		"hasTemperature":"cla:hasTemperature",
+		"atTime":"cla:atTime",
+		"hasOwner":"cla:hasOwner",
+		"hasName":"vocab:hasName"
+		
+	},
+	"@id":"http://schema.org/class01",
+	"@type":"cla:Class01",
+	"includes":[{
+		"Sensor029":{
```

Comment:
You don't need to define "Sensor029". It isn't an attribute, it is a resource, thus I suggest you remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes.

Reply:
Please, remove "Sensor029":, "Measurment8401":, "Computer101":, and "User10A": from the file. 
In addition, validate this file before resubmitting it because it has some syntax errors.